okhttp/src/main/java/okhttp3/Headers.java

```diff
@@ -334,6 +339,17 @@ public Builder addAll(Headers headers) {
       return this;
     }
 
+    /**
+     * Add a Date header with the current time based on
+     * https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Date .
+     */
+    public Builder withDate() {
+      final DateFormat format = new SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss 'GMT'", Locale.US);
```

Comment:
Call this instead? `okhttp3.internal.http.HttpDate#format`

Reply:
Sure thing